### PR TITLE
Experimential/no ref/cut off power

### PR DESCRIPTION
**Pull request by andyCLY**
Finish on feature: resume after cut off power
There is 2 notes:
+ better not to reconnect power within 10 sec, and **must not** reconnect in the same cycle
+ add super cap and connect LED to GPIO9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ The firmware for AGIS Longevity Tester
     - plz don't try to do sth strange
 - OTA can be use only after the test end, go to 192.168.0.227/update for update
 - better not to start the testing before get the time
+- ~~do not use esp.restart() to test the resume function~~
+- ~~after cut off the power supply, ~~~~DO NOT reconnect within 20 sec~~~~ DO NOT reconnect and cut off again within 20 sec~~
```
Comment (tranvnhan) [inline README.md:15]: If these are no longer true, you can remove them

File: src/TESTER_INA219.cpp [hunk 2]
```diff
@@ -30,6 +27,12 @@ void getCurrent() {
 
     // get the data from INA219
     current_mA = ina219.getCurrent_mA();
+    // mark as power failure if V<11
+    if (!powerFail && ina219.getBusVoltage_V()<10) {
+      powerFail = true;
+    } else if (powerFail && ina219.getBusVoltage_V()>10) {
+      powerFail = false;
+    }
```
Comment (tranvnhan) [inline src/TESTER_INA219.cpp:35]: Maybe use a constant or define for the voltage? Btw, comment says 11 but below is 10

File: README.md (hunk 1 above)
Comment (andyCLY) [reply README.md:15]: is still true, but as we add the super cap, we don't need the old method for testing. 
As a result, we have a rarely case that need to do that.
Just keep here as we don't know it there any ppl want to do that in the future.

Comment (tranvnhan) [reply README.md:15]: Oh, I see. Coz I saw it strikethroughed. Maybe use a note block or something. 
image

**Comment by tranvnhan** [comment]
We also need to document how the circuit is modified.
It can be an image with some annotations/explanations.

File: README.md (hunk 1 above)
Comment (andyCLY) [reply README.md:15]: interesting problem, cuz i see it works normal
image

**Comment by andyCLY** [comment]
The changes in TFT is in README already.
For super cap, its more likely to be the modification on hardware. I am going to add it to document / AT. But still, need some time

**Comment by tranvnhan** [comment]
> The changes in TFT is in README already. For super cap, its more likely to be the modification on hardware. I am going to add it to document / AT. But still, need some time

Yes, I meant the hardware changes.
Please do so next time. Thanks.

**Pull request merged**